core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1067,9 +1067,9 @@ private[spark] object Utils extends Logging {
     // finding the call site of a method.
     val SPARK_CORE_CLASS_REGEX =
       """^org\.apache\.spark(\.api\.java)?(\.util)?(\.rdd)?(\.broadcast)?\.[A-Z]""".r
-    val SCALA_CLASS_REGEX = """^scala""".r
+    val SCALA_CLASS = "scala"
```

Comment:
LGTM but could this be called `SCALA_CORE_CLASS_PREFIX`? why is a `lazy val` needed? doesn't seem worth it.